connectors/bitbucket-server/src/main/resources/cards/text.properties

```diff
@@ -1,4 +1,4 @@
-header=Bitbucket Server PR
+header=Merge Request
```

Comment:
Making the header uniform for ```bitbucket server```, ```gitlab```, ```github```.

Reply:
If you're going to make them uniform, then use the more common name (Pull Request).  Only Gitlab calls it "Merge Request".

Reply:
I have made the header suffix for ```bitbucket```, ```github``` connector as ```Pull Request``` and changed suffix of ```Gitlab``` as ```Merge Request```.